tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -85,6 +88,22 @@ pub struct Ciphertext {
     pub pbs_order: PBSOrder,
 }
 
+impl crate::Named for Ciphertext {
+    const NAME: &'static str = "shortint::Ciphertext";
+}
+
+impl ParameterSetConformant for Ciphertext {
+    type ParameterSet = CiphertextConformanceParams;
+
+    fn is_conformant(&self, param: &CiphertextConformanceParams) -> bool {
+        self.ct.as_view().is_conformant(&param.ct_params)
+            && self.message_modulus == param.message_modulus
+            && self.carry_modulus == param.carry_modulus
+            && self.pbs_order == param.pbs_order
+            && self.degree == param.degree
```

Comment:
this is good enough for our use cas for now (BC encryption) but we should perhaps think of something for a range for the degree ? wdyt ?

Reply:
I think we can add more complex constraints later when needed

Reply:
follow up issue then please :)

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/270